### PR TITLE
SPCTR-3411 : Adding background image GBS compatibility

### PR DESCRIPTION
**Pull request by rahulvarma722**
### Description
https://brainstormforce.atlassian.net/jira/software/projects/SPCTR/boards/1?assignee=62cf9daea94a6f9c0efeea37&selectedIssue=SPCTR-3411ok

### Checklist:
- [ ] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: blocks-config/uagb-controls/generateBackgroundCSS.js [hunk 1]
```diff
@@ -152,12 +152,14 @@ const generateBackgroundCSS = ( backgroundAttributes, pseudoElementOverlay = {}
 					bgCSS[ 'background-image' ] = gradient + ', url(' + backgroundImage?.url + ');';
 				}
 			}
+
 			if (
-				( 'image' === overlayType || 'none' === overlayType ) &&
+				( 'image' === overlayType || 'none' === overlayType || '' === overlayType ) &&
```
Comment (adi3890) [inline blocks-config/uagb-controls/generateBackgroundCSS.js:157]: should we have includes here

File: classes/class-uagb-block-helper.php [hunk 2]
```diff
@@ -1601,7 +1601,7 @@ public static function uag_get_background_obj( $bg_obj, $css_for_pseudo = '' ) {
 								$gen_bg_css['background-image'] = $gradient . ', url(' . $bg_img . ');';
 							}
 						}
-						if ( ( 'none' === $overlay_type || 'image' === $overlay_type ) && '' !== $bg_img ) {
+						if ( ( '' === $overlay_type || 'none' === $overlay_type || 'image' === $overlay_type ) && '' !== $bg_img ) {
```
Comment (adi3890) [inline classes/class-uagb-block-helper.php:1604]: should we try in_array here

**Review by adi3890 — APPROVED** [review]

File: blocks-config/uagb-controls/generateBackgroundCSS.js (hunk 1 above)
Comment (rahulvarma722) [reply blocks-config/uagb-controls/generateBackgroundCSS.js]: Done

File: classes/class-uagb-block-helper.php (hunk 2 above)
Comment (rahulvarma722) [reply classes/class-uagb-block-helper.php]: Done

**Review by adi3890 — APPROVED** [review]

**Pull request merged**